libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4999,11 +4999,9 @@ MAV_RESULT GCS_MAVLINK::handle_command_int_external_position_estimate(const mavl
         return MAV_RESULT_DENIED;
     }
     uint32_t timestamp_ms = correct_offboard_timestamp_usec_to_ms(uint64_t(p2.param1*1e6), PAYLOAD_SIZE(chan, COMMAND_INT));
-    const uint32_t processing_ms = p2.param2*1e3;
+    const uint32_t processing_delay_ms = (uint32_t)MAX(p2.param2*1e3f, 0.0f);
```

Comment:
i don't understand, the original code does treat it as a delay, and subtracts it. Not sure what you are fixing here